deepcell/applications/pannuke_nuclear_seg.py

```diff
@@ -0,0 +1,127 @@
+"""Nuclear segmentation application for Pannuke Dataset"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import tensorflow as tf
+
+from deepcell_toolbox.processing import normalize
+from deepcell_toolbox.deep_watershed import deep_watershed
+
+from deepcell.applications import Application
+
+
+MODEL_PATH = ('./pannuc_panopticnet.h5')
+
+
+class NuclearSegmentation(Application):
```

Comment:
This name must be updated, maybe `PanNukeSegmentation`. Be sure to update the docstring also.

The name of the file can also be updated to `pannuke_segmentation.py` for more clarity.